test/python/quantum_info/operators/test_measures.py

```diff
@@ -64,7 +64,7 @@ def test_channel_process_fidelity(self):
         # Depolarizing channel
         prob = 0.5
         op = Operator.from_label('Y')
-        chan = (prob * depol + (1 - prob) * iden) @ op
+        chan = (prob * depol + (1 - prob) * iden).compose(op)
```

Comment:
In the same way that [NumPy advocates for the use of the operators instead of methods](https://numpy.org/doc/stable/reference/generated/numpy.dot.html), should we do the same and advocate `&` as a shorthand for `.compose()`?

Reply:
Something like that should probably be done once we are consistent with operators across different classes (operators, circuits, opflow etc).